### DISCUSSION TITLE
Cannot find the Azure DevOps pipeline for the CUDA 10.2 build

### DISCUSSION THREAD

**Discussion by ramteid** [microsoft/onnxruntime#7862] [Other Q&A]
I'm looking for a build of onnxruntime 1.7.x with support for CUDA 10.2. I found this YAML:
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/win-gpu-cuda-10-2-pipeline.yml

Which of your pipelines uses this YAML?

**Comment by snnn** [comment]
The pipeline is not at https://dev.azure.com/onnxruntime/onnxruntime/_build , we don't use it for gating our PRs.  Anything else you want to know?
